src/sitemap/handler.js

```diff
@@ -393,8 +394,78 @@ export async function sitemapAuditRunner(baseURL, context) {
   };
 }
 
+async function convertToOpportunity(auditUrl, auditData, context) {
+  const { dataAccess, log } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let sitemapOpptyPagesWithIssues = opportunities.find((oppty) => oppty.getType() === 'sitemap-pages-with-issues');
```

Comment:
shouldn't the opportunity type be agnostic of whether the sitemap doesn't-exist or sitemap pages have error